### PR TITLE
Update for readme

### PR DESCRIPTION
**Pull request by nloadholtes**
Hi, just added some details for the readme. Please feel free to change, etc. Thanks!

Links to the docs, etc.

JIRA: 

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit 516fb046958d3b8aeae78bbcfd414f5d86e62071

https://app.netlify.com/sites/helix-ui/deploys/5bc8aeacc965923565b04815

**Comment by CITguy** [comment]
@nloadholtes: thanks for submitting this PR. We're a little swamped right now, but we'll try to review this PR as soon as we have bandwidth.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+[Homepage](https://rackerlabs.github.io/helix-ui/)
+
+## Documentation
+ * [Getting Started](https://rackerlabs.github.io/helix-ui/guides/getting-started/)
+
+ * [React Compatiblity](https://rackerlabs.github.io/helix-ui/guides/react-compatibility/)
+
+ * [Components](https://rackerlabs.github.io/helix-ui/components/accordions/)
+ 
+ * [Elements](https://rackerlabs.github.io/helix-ui/elements/hx-accordion/)
```
Comment (CITguy) [inline README.md]: Let's exclude the "Components" and "Elements" links, for now. (lines 11-13)
We don't have landing pages for those sections, yet.

**Comment by nloadholtes** [comment]
Thanks for looking at it! I went ahead and updated with the requested changes.

**Comment by CITguy** [comment]
Merging, even though SauceLabs insists on failing the build.

**Pull request merged**